internal/pkg/api/storageapi/config.go

```diff
@@ -123,8 +123,13 @@ func (c ConfigMetadataResponse) MetadataMap(branchId model.BranchId) map[model.C
 	return result
 }
 
-// UpdateConfigMetadataRequest https://keboola.docs.apiary.io/#reference/metadata/components-configurations-metadata/create-or-update
-func (a *Api) UpdateConfigMetadataRequest(config *model.Config) *client.Request {
+// AppendConfigMetadataRequest https://keboola.docs.apiary.io/#reference/metadata/components-configurations-metadata/create-or-update
```

Comment:
Premenoval som to na `Append`, kedze nove hodnoty sa pridaju/prepisu, stare ostanu.